docs/community/governance.md

```diff
@@ -83,37 +92,70 @@ For all votes, voting must be open for at least one week. The end date should be
 
-## Supermajority vote
+### Supermajority vote
 
-Supermajority votes must be called explicitly in a separate thread on the appropriate mailing list. The subject must be prefixed with [VOTE]. In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
+Supermajority votes must be called explicitly in a separate thread on the appropriate mailing list. The subject must be prefixed with `[VOTE]`. In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
 
 Votes may take the form of a single proposal, with the option to vote yes or no, or the form of multiple alternatives.
 
 A vote on a single proposal is considered successful if at least two thirds of those eligible to vote vote in favor.
 
 If there are multiple alternatives, members may vote for one or more alternatives, or vote “no” to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+## On- / Offboarding
+
+### Onboarding
+
+The new member is
+
+* added to the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR.
+* announced on the [developers mailing list][devs] by an existing team member. Ideally, the new member replies in this thread, acknowledging team membership.
+* added to the projects with commit rights.
+* added to the [team mailing list][team].
+
+
+### Offboarding
+
+The ex-member is
+
+* removed from the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR. In case of forced removal, no approval is needed.
+* removed from the projects. Optionally, they can retain maintainership of one or more repositories if the [team](#team-members) agrees.
+* removed from the team mailing list and demoted to a normal member of the other mailing lists.
+* not allowed to call themselves an active team member any more, nor allowed to imply this to be the case.
+* added to a list of previous members if they so choose.
+
+If needed, we reserve the right to publicly announce removal.
+
+[announce]: https://groups.google.com/forum/#!forum/loki-announce
+[coc]: https://github.com/grafana/loki/blob/master/CODE_OF_CONDUCT.md
+[devs]: https://groups.google.com/forum/#!forum/loki-developers
+[maintainers]: https://github.com/grafana/loki/blob/master/MAINTAINERS.md
```

Comment:
This file doesn't exist I guess

Reply:
Correct, @slim-bean needs to decide if that's any subsystem maintainers or if he wants to be listed for `*`.

Reply:
To start we won't assign any subsystem maintainers, however I expect this to change over time.

I think you are stuck with me for all the things :)

Reply:
ALL THE THINGS!!!111